src/gui/attributetable/qgsdualview.cpp

```diff
@@ -865,11 +865,22 @@ void QgsDualView::showViewHeaderMenu( QPoint point )
   connect( setWidth, &QAction::triggered, this, &QgsDualView::resizeColumn );
   setWidth->setData( col );
   mHorizontalHeaderMenu->addAction( setWidth );
+
+  QAction *setWidthAllColumns = new QAction( tr( "&Set Width All..." ), mHorizontalHeaderMenu );
```

Comment:
I don't particularly like the sentence fragment here, maybe:
```suggestion
  QAction *setWidthAllColumns = new QAction( tr( "&Set All Column Widths…" ), mHorizontalHeaderMenu );
```
is better? (also note the use of the typographically correct "…" character instead of the 3 dots "..." :+1: )

Reply:
Thanks, yep that looks much better.